buildstockbatch/gcp/gcp.py

```diff
@@ -577,7 +581,7 @@ def start_batch_job(self, batch_info):
         # Give three minutes per simulation, plus ten minutes for job overhead
         task_duration_secs = 60 * (10 + batch_info.n_sims_per_job * 3)
         task = batch_v1.TaskSpec(
-            runnables=[runnable],
+            runnables=[runnable, prune],
```

Comment:
Now that there are two runnables, can we rename `runnable` to something like `bsb_runnable`?

And maybe a better name than `prune` might be `cleanup_runnable` (hide the implementation detail) or `docker_prune_runnable` (qualify what's being pruned). I lean towards the former, fwiw.

Reply:
Agreed - this was a quick version to start testing with, but I'll clean it up before merging.